### PR TITLE
Add integration tests for prf results table

### PR DESCRIPTION
**Pull request by yuki617**

### CODE REVIEW

File: integrations/simplesearcher_score_checker.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class SimpleSearcherScoreChecker:
+    def __init__(self, index: str, topics: str, pyserini_topics: str, qrels: str, eval:str):
+        self.index_path = index
+        self.topics = topics
+        self.qrels = qrels
+        self.pyserini_topics = pyserini_topics
+
+        self.pyserini_base_cmd = 'python3 -m pyserini.search'
```
Comment (lintool) [inline integrations/simplesearcher_score_checker.py:30]: I think this should just be `python` not `python3`?

File: integrations/test_simplesearcher_check_prf.py [hunk 2]
```diff
@@ -0,0 +1,661 @@
...
+            pyserini_topics='core18',
+            qrels=os.path.join(self.pyserini_root, 'tools/topics-and-qrels/qrels.core18.txt'),
+            eval=f'{self.pyserini_root}/tools/eval/trec_eval.9.0.4/trec_eval -m map -m P.30')
+
+        if os.path.exists(self.tmp):
+            shutil.rmtree(self.tmp)
+        else:
+            os.mkdir(self.tmp)
+
+        download_url('https://www.dropbox.com/sh/xo9v2glmwy3jrtn/AADdx7gdes6_pPAQKDPjgWDSa/core18.tar.gz?dl=1',
```
Comment (lintool) [inline integrations/test_simplesearcher_check_prf.py:53]: Remove - assume index is there, throw exception if it isn't...

File: integrations/test_simplesearcher_check_prf.py [hunk 3]
```diff
@@ -0,0 +1,661 @@
...
+    # Core17
+
+    def test_core17(self):
+        self.assertTrue(self.checker17.run('core17_bm25', '--bm25', 0.2087))
+
+    def test_core17_rm3(self):
+        self.assertTrue(self.checker17.run('core17_bm25', '--bm25 --rm3',0.2823))
+
+    def test_core17_lr(self):
+        os.system(f'python3 {self.pyserini_root}/scripts/classifier_prf/cross_validate.py --anserini {self.anserini_root} \
```
Comment (lintool) [inline integrations/test_simplesearcher_check_prf.py:116]: just `python`.

File: integrations/test_simplesearcher_check_prf.py (hunk 2 above)
Comment (yuki617) [reply integrations/test_simplesearcher_check_prf.py]: Try and exception added. Python3 is also updated to python.

**Review by lintool — APPROVED** [review]

**Pull request merged**